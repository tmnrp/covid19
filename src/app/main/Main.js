import "./Main.scss";
import React from "react";

const Main = () => {
  return <div data-testid="main-testid">Main</div>;
};

export default Main;
