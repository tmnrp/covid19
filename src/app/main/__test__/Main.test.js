import React from "react";
import Main from "../Main";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

test("Footer should render", () => {
  const { getByTestId } = render(<Main />);
  expect(getByTestId("main-testid")).toMatchSnapshot();
});
