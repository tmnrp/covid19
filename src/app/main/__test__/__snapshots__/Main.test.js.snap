// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<div
  data-testid="main-testid"
>
  Main
</div>
`;
